data/human/human missions.txt

```diff
@@ -4077,3 +4077,93 @@ mission "Laborer Payment"
 		conversation
 			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
 				decline
+
+
+
+mission "Block Hai Wormhole Warning"
+	landing
+	invisible
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		has "chosen sides"
+		not "main plot completed"
+	on offer
+		fail
+
+mission "Hai Wormhole Warning"
+	minor
+	source
+		government "Republic"
+		not attributes "deep"
+	to offer
+		or
+			has "First Contact: Hai"
+			has "First Contact: Unfettered"
+		not "Block Hai Wormhole Warning: offered"
+		not "Wanderers: Alpha Surveillance E: offered"
+	on offer
+		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
+		conversation
+			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
+			choice
+				`	"What about?"`
+					goto what
+				`	"Okay."`
+					goto frigate
+				`	"Sorry, I'm really busy right now."`
+			`	"No, I'm sorry, but this isn't a matter of whether you're busy or not right now. Come with me."`
+				goto frigate
+			label what
+			`	"I did say in private, did I not?" he says, once again motioning to the Frigate.`
+			label frigate
+			`	The man puts a hand on your shoulder and begins walking you toward the Frigate. As you approach, you note that the man's uniform doesn't have any Naval markings that you recognize.`
+			`	You enter the Frigate with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
```

Comment:
```suggestion
			`	You enter the Frigate with the officer, and despite the decent crew requirement of that ship, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
```

Reply:
How about flipping which "Frigate" we replace, since the previous line already notes that it's a Frigate.
```suggestion
			`	You enter the ship with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
```